metacatalog/ext/standards_export/test_export.ipynb

```diff
@@ -0,0 +1,572 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from metacatalog import api, ext\n",
```

Comment:
I would like to remove this notebook from the repo and add it to the gitignore. Or is there any specific reason, whiy the user would need it?

Reply:
I pushed that by accident, thanks for pointing this out!